contributors/guide/release-notes.md

```diff
@@ -16,20 +16,14 @@ For pull requests with a release note:
     Your release note here
     ```
 
-For pull requests that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
+For pull requests that require additional action from users switching to the new release, write your Release Note as usual and add the `release-note-action-required` label to the PR.
 
-    ```release-note
-    action required: your release note here
-    ```
-
-For pull requests that don't need to be mentioned at release time, just write "NONE" (case insensitive):
+For pull requests that don't need to be mentioned at release time, use the `/release-note-none` Prow command to add the `release-note-none` label to the PR. You can also write the string "NONE" as a release note in your PR description:
 
     ```release-note
     NONE
     ```
 
-The `/release-note-none` comment command can still be used as an alternative to writing "NONE" in the release-note block if it is left empty.
-
-To see how to format your release notes, view the kubernetes/kubernetes [pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. pull request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
+To see how to format your release notes, view the kubernetes/kubernetes [pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. Pull Request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
 
-Release notes apply to pull requests on the master branch. For cherry-pick pull requests, see the [cherry-pick instructions](contributors/devel/cherry-picks.md). The only exception to these rules is when a pull request is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master pull request.
\ No newline at end of file
+Release notes apply to pull requests on the master branch. For cherry-pick pull requests, see the [cherry-pick instructions](contributors/devel/cherry-picks.md). The only exception to these rules is when a pull request is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master pull request.
```

Comment:
nit: modify the relative link to `/contributors/devel/cherry-picks.md`?

Reply:
> In this case, a `release-note-*` label is required for that non-master pull request.

This would mean the non-cherry-pick PR to a non-master branch can have `release-note-none`? I think it makes sense to enforce that it should have a release note. Wdyt?

Reply:
I wonder...are all post-release patches on a release branch implicitly not 'release-note-none'?  If so, it would be good to not burden the release patch manager with manually enforcing and instead have automation that makes it clear to the cherry pick proposer that a note is required.

Reply:
/cc @feiskyer @foxish @MaciekPytel @mbohlool 

I'm curious your thoughts on this.